test/app/presenters/track_test.rb

```diff
@@ -7,20 +7,34 @@ class PresentersTrackTest < Minitest::Test
   def setup
     @track = ExercismWeb::Presenters::Track.new("test_track")
     @trackler_track = Object.new
+
     @track.stubs(:trackler_track).returns(@trackler_track)
     @track.stubs(:fallback_topic_content).returns("Default topic content")
```

Comment:
I realise this is not your fault, but I really don't like stubbing the object under test.
`tracker_track` could be tested by stubbing `Trackler` instead.
Is `fallback_topic_content` ever tested?